app/components/FeedPost.tsx

```
@@ -1,72 +1,83 @@
                 src="/icons/messages.svg"
                 alt="View message thread"
                 width={20}
                 height={20}
               />
-            )}
-          </p>
+            </button>
+          </div>
           <div
             className={styles.postContent}
             dangerouslySetInnerHTML={{ __html: content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)